src/commands/deployWorkspaceProject/deployWorkspaceProject.ts

```diff
@@ -37,8 +39,13 @@ export async function deployWorkspaceProject(context: IActionContext & Partial<D
         subscription
     });
 
+    ext.outputChannel.appendLog(
+        formatSectionHeader(localize('preparingDeploymentConfiguration', 'Preparing workspace deployment configuration'))
```

Comment:
Should we just start using the new localize package? It'll make it easier to convert, but also I find it annoying to always have to come up with a "localize title key" for each string we have.

Reply:
Yeah, I agree, I hate having to come up with new id's all the time.  Maybe an item for next release